src/__init__.py

```diff
@@ -16267,15 +16267,15 @@ def JM_irect_from_py(r):
         r = mupdf.FzIrect( r.x0, r.y0, r.x1, r.y1)
         return r
     if isinstance(r, Rect):
-        ret = mupdf.fz_make_irect(r.x0, r.y0, r.x1, r.y1)
+        ret = mupdf.fz_make_irect(int(r.x0), int(r.y0), int(r.x1), int(r.y1))
```

Comment:
I don't think this is correct, because rounding is different from how MuPDF makes a IRect from a Rect: uppper and left coords must be rounded in direction left / up, but lower / right coords must take the next available integer ceiling.

Reply:
Thanks, i hadn't realised that. New commit fixes hopefully.